diff --git a/sk-usbhid.c b/sk-usbhid.c
index 7bb829aa..85c027a1 100644
--- a/sk-usbhid.c
+++ b/sk-usbhid.c
@@ -75,10 +75,10 @@
 #define FIDO_CRED_PROT_UV_OPTIONAL_WITH_ID 0
 #endif
 
+#include "misc.h"
 #ifndef SK_STANDALONE
 # include "log.h"
 # include "xmalloc.h"
-# include "misc.h"
 /*
  * If building as part of OpenSSH, then rename exported functions.
  * This must be done before including sk-api.h.
